src/preact/base-element.js

```diff
@@ -35,6 +34,8 @@ import {HAS_SELECTOR, checkPropsFor, collectProps} from './parse-props';
 
 /** @typedef {import('./parse-props').AmpElementProp} AmpElementProp */
 
+/** @typedef {{readyState?: ReadyState_Enum, pause?: function():void }} API_TYPE */
```

Comment:
I don't think this is right. `API_TYPE` is a generic (template) type and does not make sense on its own.